lincs/src/org/labkey/lincs/LincsController.java

```diff
@@ -1065,6 +1073,112 @@ public void setApiKey(String apiKey)
         }
     }
 
+    @RequiresPermission(AdminPermission.class)
+    public class CromwellConfigAction extends FormViewAction<CromwellConfigForm>
+    {
+        @Override
+        public void validateCommand(CromwellConfigForm target, Errors errors) {}
+
+        @Override
+        public boolean handlePost(CromwellConfigForm form, BindException errors)
+        {
+            CromwellConfig config = form.getConfig();
+            try
+            {
+                config.validate();
+            }
+            catch (CromwellException e)
+            {
+                errors.reject(ERROR_MSG, e.getMessage());
```

Comment:
Will there always be a message in this exception? If not, you should include some sort of generic error so we have something to show to the user